src/Nox.Reference.MacAddresses/Services/IMacAddressService.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Nox.Reference.Abstractions.MacAddresses;
+
+namespace Nox.Reference.MacAddresses
+{
+    public interface IMacAddressService
+    {
+        /// <summary>
+        /// Get all mac addresses with vendors.
+        /// </summary>
+        /// <returns>Collection of IMacAddressInfo</returns>
+        IReadOnlyList<IMacAddressInfo> GetMacAddresses();
```

Comment:
I know we must still spec the API interface, but all services should follow `Get{ServiceName-Singular}Info` pattern. Eg.

- GetPhoneNumberInfo
- GetCountryInfo
- GetMacAddressInfo
- ...etc